rosidl_typesupport_c/include/rosidl_typesupport_c/type_support_map.h

```diff
@@ -22,15 +22,26 @@ extern "C"
 {
 #endif
 
+/// Standard structure for encapsulating information about a rosidl interface type's support.
```

Comment:
> Standard structure for

This doesn't add any value to the docstring.

> encapsulating information about a rosidl interface type's support

This isn't very helpful either. Maybe something like "contains all available C typesupport handles to choose from".